integration/tests/cook/util.py

```diff
@@ -51,8 +51,16 @@ def retrieve_cook_url(varname='COOK_SCHEDULER_URL', value='http://localhost:1232
     return cook_url
 
 
-def retrieve_mesos_url(varname='MESOS_URL', value='http://localhost:5050'):
-    mesos_url = os.getenv(varname, value)
+def retrieve_mesos_url(varname='MESOS_PORT', value='5050'):
+    mesos_port = os.getenv(varname, value)
+    cook_url = retrieve_cook_url()
+    wait_for_cook(cook_url)
+    settings = session.get('%s/settings' % cook_url).json()
```

Comment:
There is a `settings` function in `test_basic.py`. Might be nice to move it into `util` and use it here.